### ISSUE TITLE
Game stuck in 1024x768, Won't allow other option

### ISSUE THREAD

**Issue by Angel0A** [jdmclark/jkgfxmod#56]
Hi. I used the game installer from Mod DB 'Jedi Knight Remastered v1.0'

After some tweaking I have the game running well. However, I have no option to change the resolution. 

In-game menu only allows 1024x768.

My native resolution is 1920x1024. Editing the .json file does nothing, the game will simply not launch.

Using win 10 with Radeon 5500XT

**Comment by Angel0A** [comment]
Heeeey

**Comment by nikumubeki** [comment]
Are you running inject.exe (possibly also jk.exe) in admin mode? 

On my setup, for some reason, I have to run both jk.exe and inject.exe in admin mode to even get the game to launch, so that might be worth trying.

**Comment by Angel0A** [comment]
Yes! I run ("...\Star Wars Jedi Knight\inject.exe" -windowgui -devmode) in admin mode

**Comment by Angel0A** [comment]
My game has no problem launching, it's just stuck running in the smaller resolution.

**Comment by nikumubeki** [comment]
Hmm, I don't think you need to use -windowgui and -devmode with inject.exe, but I don't think those should affect the resolution in any case... : \

**Comment by jdmclark** [comment]
Hi @ValkyrieRaregod 

- Do you get any sort of error dialog when the game fails to launch? If so, what does the dialog box say?
- Try setting your resolution in the config file, and set `fullscreen` to `false`. Does the game launch?
- Please post a log of the game failing to launch. See the main README for instructions.

Thanks
